docs/guides/architecture/_category_.yml

```diff
@@ -1 +1,3 @@
 position: 3
```

Comment:
Do you want to add the title to this one explicitly too? You did it with all others so maybe?

Reply:
The title is only really needed if the default capitalisation isn't right or you don't have a README page (otherwise it pulls the title from the readme page